src/ui/username-label/username-label.tsx

```diff
@@ -0,0 +1,21 @@
+import { UsernameLabelWrapper } from './username-label.styles';
+import {UsernameLabel} from './username-label.styles';
+import { CSSProperties } from 'react'; 
+
+type UserNameLabelProps = {
+  username: string;
+  styles?: CSSProperties;
+};
+
+export const UserNameLabel: React.FC<UserNameLabelProps> = (props: UserNameLabelProps) => {
+
+    const shortName = props.username.split(' ')[0].charAt(0) + props.username.split(' ')[1].charAt(0);

```

Comment:
A short form:

```typescript
const [[firstLetter = ''] = '', [secondLetter = ''] = ''] = props.username.split(' ');
```

just FYI; feel free to use the code that you understand better;

also check pls the case when username is not contain ` `; for now your code will be broken when passing, e.g., 'Somename`